### ISSUE TITLE
Can't install litmus with the namespace scope

### ISSUE THREAD

**Issue by rberrelleza** [litmuschaos/litmus-helm#164]
When I try to install litmus 2.0.32 using a namespace-scoped account, the installation fails.

This is because the role we use asks for [`nodes` and `namespaces` permissions](https://github.com/litmuschaos/litmus-helm/blob/master/charts/litmus/templates/server-role.yaml#L143), which is not namespace scoped.

**Comment by rberrelleza** [comment]
What is the downside of not having those permissions?

**Comment by rajdas98** [comment]
Yes, these are some extra permission. I will raise a PR to remove it. Thanks for pointing this.

Meanwhile, you can use this k8s manifest in namespace mode- https://github.com/litmuschaos/litmus/blob/master/mkdocs/docs/2.0.0/litmus-namespaced-2.0.0.yaml

**Comment by uditgaurav** [comment]
Hi @rberrelleza , We have already removed the higher permission in namespace scoped service account. Thanks for opening it.
